.github/workflows/test/linux-nightly-ci.yml

```
@@ -0,0 +1,101 @@
+  main-nightly:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: lts/*
+      - name: Install vscode dependencies
+        run: sudo apt-get update && sudo apt-get install -y wget
+
+      - uses: actions/setup-java@v3
```

actionlint (1.7.7)

the runner of "actions/setup-java@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)